src/css/index.scss

```diff
@@ -0,0 +1,105 @@
+section.contact {
+  padding-bottom: 100px;
+  padding-top: 10px;
+}
+section.contact h2 {
+  font-size: 38px;
+  font-size-adjust: inherit;
+  font-family: avenir, sans-serif;
+  font-style: normal;
+  margin-bottom: 2%;
+  margin-top: 20px;
+  padding-left: 10%;
+}
+div.red_bar {
+  width: 70px;
+  margin-bottom: 2%;
+  border-top: 7px solid #df3143;
+  margin-left: 10%;
+}
+
+section#quick_id {
+  float: left;
+  color: white;
+  background-color: #2e4975;
+  width: 50%;
+  height: 300px;
+}
+section#contact_me {
+  float: right;
+  background-color: lightsteelblue;
+  width: 50%;
+  height: 300px;
+}
+
+table {
+  padding-left: 10%;
+  text-align: left;
+  font-size: 20px;
+  font-family: sans-serif;
+}
+table td {
+  padding-left: 20%;
```

Comment:
Using padding left with percentage is not recommended. You will never get close to pixel perfect as design since the padding is based on container width.